pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+
+  pname = "sleek-grub-theme";
+  version = "unstable-2022-06-04";
+
+  src = fetchFromGitHub ({
+    owner = "sandesh236";
+    repo = "sleek--themes";
+    rev = "981326a8e35985dc23f1b066fdbe66ff09df2371";
+    hash = "sha256-yD4JuoFGTXE/aI76EtP4rEWCc5UdFGi7Ojys6Yp8Z58=";
+  });
+
+  banner = "Grub Bootloader"; # can override to your own banner
+
+  style = "white"; # can override to "white" / "dark" / "orange" / "bigSur"
```

Comment:
```suggestion
```

Reply:
Now using `stdenv.mkDerivation (finalAttrs: { ... })` instead of `rec` to solve the `overrideAttrs` issue